### ISSUE TITLE
[BUG] False positive for font-semibold tailwind utility class

### ISSUE THREAD

**Issue by AndyOGo** [francoismassart/eslint-plugin-tailwindcss#151]
**Describe the bug**

`font-semibold` is an official Tailwind V3 font-weight utility:
https://tailwindcss.com/docs/font-weight

But code like this:

```jsx
<div className="font-semibold" />
```

produces this warning:

> warning  Classname 'font-semibold' is not a Tailwind CSS class!  tailwindcss/no-custom-classname

**Expected behavior**
No warning.

**Environment (please complete the following information):**
 - OS: Redhat Linux
 - Softwares + version used:
    ```sh
    $ node -v
    v16.13.0
    $ npm -v
    8.1.0
    ```

**Additional context**

**eslint config file or live demo**
```js
module.exports = {
  parserOptions: {
    ecmaFeatures: {
      jsx: true,
    },
  },
  plugins: ["tailwindcss"],
  settings: {
    tailwindcss: {
      whitelist: ["_.+", "e2e-.+"],
    },
  },
  rules: {
    "tailwindcss/classnames-order": "error",
    "tailwindcss/enforces-negative-arbitrary-values": "error",
    "tailwindcss/enforces-shorthand": "error",
    "tailwindcss/migration-from-tailwind-2": "warn",
    "tailwindcss/no-arbitrary-value": "off",
    "tailwindcss/no-custom-classname": "warn",
    "tailwindcss/no-contradicting-classname": "error",
  },
};
```

**Comment by AndyOGo** [comment]
Sorry, that was caused by a preset file.
